platform-controller/src/main/java/org/hobbit/controller/docker/ClusterManagerImpl.java

```diff
@@ -0,0 +1,57 @@
+package org.hobbit.controller.docker;
+
+import com.spotify.docker.client.DockerClient;
+import com.spotify.docker.client.exceptions.DockerCertificateException;
+import com.spotify.docker.client.exceptions.DockerException;
+import com.spotify.docker.client.messages.Info;
+import org.hobbit.controller.ExperimentManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * ClusterManager implementation
+ *
+ * @author Ivan Ermilov (iermilov@informatik.uni-leipzig.de)
+ *
+ */
+public class ClusterManagerImpl implements ClusterManager {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClusterManagerImpl.class);
+
+    /**
+     * Docker client instance
+     */
+    private DockerClient dockerClient;
+    private Integer expectedNumberOfNodes = 0;
+    private String SWARM_NODE_NUMBER = null;
+
+    public ClusterManagerImpl() throws DockerCertificateException {
+        dockerClient = DockerUtility.getDockerClient();
```

Comment:
I think the platform components should share one single docker client instance. Otherwise, if you want to configure the client, you will have to add the configuration to a lot of classes.